pkg/server/io/helpers.go

```diff
@@ -92,3 +92,13 @@ type stdioPipes struct {
 	stdout io.ReadCloser
 	stderr io.ReadCloser
 }
+
+func newEmptyPipes() (*stdioPipes, *wgCloser) {
+	ctx, cancel := context.WithCancel(context.Background())
+	return &stdioPipes{}, &wgCloser{
+		wg:     &sync.WaitGroup{},
+		set:    make([]io.Closer, 0),
```

Comment:
Why this instead of just `[]io.Closer{}`?

Reply:
good point 🤦‍♂️